### PR TITLE
refactor: code cleanup phase 1

### PR DESCRIPTION
**Pull request by beyackle**
## Description

Phase one of some miscellaneous cleanup on our codebase. I'm starting with the `client/src/` directory and going through there alphabetically, which is why this contains what it does. The point here is just refactoring with no functional changes.

## Task Item

refs #3307

### CODE REVIEW

File: Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx [hunk 1]
```diff
@@ -45,7 +44,7 @@ interface DefineConversationProps
 
 const DefineConversation: React.FC<DefineConversationProps> = (props) => {
   const { onSubmit, onDismiss, onCurrentPathUpdate, saveTemplateId, templateId, focusedStorageFolder } = props;
-  const files = get(focusedStorageFolder, 'children', []);
+  const files = focusedStorageFolder.children ?? [];
```
Comment (a-b-r-o-w-n) [inline Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx:47]:
```suggestion
  const files = focusedStorageFolder?.children ?? [];
```

File: Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx [hunk 2]
```diff
@@ -72,17 +70,15 @@ const DefineConversation: React.FC<DefineConversationProps> = (props) => {
         }
 
         const newBotPath =
-          focusedStorageFolder && Object.keys(focusedStorageFolder as Record<string, any>).length
+          focusedStorageFolder != null && Object.keys(focusedStorageFolder as Record<string, any>).length
```
Comment (a-b-r-o-w-n) [inline Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx:73]:
```suggestion
          focusedStorageFolder !== null && Object.keys(focusedStorageFolder as Record<string, any>).length
```

File: Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx (hunk 1 above)
Comment (beyackle) [reply Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx:47]: Will do.

File: Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx (hunk 2 above)
Comment (beyackle) [reply Composer/packages/client/src/components/CreationFlow/DefineConversation/index.tsx:73]: Good catch. Will fix.

**Pull request merged**